SPECS/python3/python3.spec

```diff
@@ -163,13 +151,7 @@ The test package contains all regression tests for Python as well as the modules
 
 # Ideally we would use '%%autopatch -p1 -M 999', but unfortunately the GitHub CI pipelines use a very old version of rpm which doesn't support it.
```

Comment:
I think we upgraded `rpm` and was wondering if we should re-evaluate to use `autopatch`  here ?

Reply:
+1. I'm curious if we can now drop the `-N` flag for `%autosetup` and remove manual application of the patches.

Reply:
yes, it looks like we can drop this